front/app/containers/InitiativesNewPage/InitiativesNewFormWrapper.tsx

```diff
@@ -20,7 +20,7 @@ interface Props {
 }
 
 const InitiativesNewFormWrapper = (_props: Props) => {
-  const { mutate: addInitiative } = useAddInitiative();
+  const { mutateAsync: addInitiative } = useAddInitiative();
```

Comment:
[Needed](https://citizenlabco.slack.com/archives/CQAGMHKT3/p1694597234083299?thread_ts=1694596416.404849&cid=CQAGMHKT3) for the error to be caught.